kernel/kernbench.py

```diff
@@ -120,8 +120,11 @@ def setUp(self):
                 self.cancel('%s is needed for the test to be run' % package)
         self.kernel_version = platform.uname()[2]
         self.iterations = self.params.get('runs', default=1)
+        self.online_cpu = 2 * cpu.online_cpus_count()
         self.threads = self.params.get(
-            'cpus', default=2 * cpu.online_cpus_count())
+            'cpus', default=self.online_cpu)
+        if self.threads is None:
```

Comment:
i feel yaml input is user defined input so handling that in script not seems me valid case  , it can be user error

Reply:
@PraveenPenguin yes, I partly agree with this. I would prefer we can have something like 
```
        self.threads = self.params.get(
            'cpus', default=None)
        if self.threads is None:
            self.threads = self.online_cpu
```
so that it takes care of empty/null inputs from yaml also.

Reply:
I have done the changes. Please have a look into it